<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>521</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(118, 214, 255);</string>
   </property>
   <property name="title">
    <string>Login</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:blue;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_user"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: blue;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_2">
      <property name="styleSheet">
       <string notr="true">color: blue;
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 64, 255);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="statusbar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 64, 255);</string>
   </property>
   <property name="text">
    <string>Loading..</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
